pandas/io/tests/parser/common.py

```diff
@@ -1467,5 +1469,23 @@ def test_memory_map(self):
             'c': ['I', 'II', 'III']
         })
 
-        out = self.read_csv(mmap_file, memory_map=True)
-        tm.assert_frame_equal(out, expected)
+        result = self.read_csv(mmap_file, memory_map=True)
+        tm.assert_frame_equal(result, expected)
+
+    def test_read_csv_utf_aliases(self):
+        # see gh issue 13549
+        path = 'test.csv'
+        expected = DataFrame({'A': [0, 1], 'B': [2, 3],
```

Comment:
1) We like to have tests that are as compact as possible.  Do we really need to have this many rows for this test?  Can we get away with just one?

2) To make these tests as unit-like as possible, we would like to NOT use `to_csv` and follow the `StringIO(data)` paradigm.  I believe that is possible here because you can encode strings as `utf-8` or `utf-16`.